react/src/DatePicker/DatePickerContext.tsx

```diff
@@ -202,11 +199,7 @@ const useProvideDatePicker = ({
 
   const handleInputChange = useCallback(
     (event: React.ChangeEvent<HTMLInputElement>) => {
-      const date = parse(
-        event.target.value,
-        dateFormat,
-        new Date(),
-      )
+      const date = parse(event.target.value, dateFormat, new Date())
```

Comment:
nit: seems like a lint change? there should be no changes here, is your linter interfering with this? may need npm i to use this repo's lint rules